ArduCopter/Log.cpp

```diff
@@ -484,18 +502,82 @@ const struct LogStructure Copter::log_structure[] = {
       "DU32",  "QBI",         "TimeUS,Id,Value", "s--", "F--" },
     { LOG_DATA_FLOAT_MSG, sizeof(log_Data_Float),         
       "DFLT",  "QBf",         "TimeUS,Id,Value", "s--", "F--" },
+    
+// @LoggerMessage: HELI
+// @Description: Helicopter related messages 
+// @Field: TimeUS: Microseconds since system startup
+// @Field: DRRPM: Desired rotor speed
+// @Field: ERRPM: Estimated rotor speed when no speed sensor or governor is used
+// @Field: Gov: Governor Output
+// @Field: Throt: Logic controlled output
```

Comment:
I'd worry if this wasn't logic-controlled :-)  Best include the word "throttle" here somewhere